packages/lesswrong/components/seasonal/HomepageMap/HomepageCommunityMap.tsx

```diff
@@ -134,15 +134,16 @@ const LocalEventMapMarkerWrappersComponent = registerComponent("LocalEventMapMar
 });
 
 
-export const HomepageCommunityMap = ({classes}: {
+export const HomepageCommunityMap = ({dontAskUserLocation = false, classes}: {
+  dontAskUserLocation?: boolean,
   classes: ClassesType,
 }) => {
   const { LocalEventMapMarkerWrappers, HomepageMapFilter } = Components
 
   const currentUser = useCurrentUser()
  
   // this is unused in this component, but for Meetup Month it seems good to force the prompt to enter location.
-  useUserLocation(currentUser)
+  useUserLocation(currentUser, dontAskUserLocation)
```

Comment:
@b0b3rt I think you should just remove `useUserLocation()` from this component - it seems like it's not doing anything anymore?